model/map-jpa/src/main/resources/META-INF/users/jpa-users-changelog-2.xml

```diff
@@ -39,4 +39,36 @@ limitations under the License.
         <addUniqueConstraint tableName="kc_user" columnNames="realmid, emailconstraint"/>
     </changeSet>
 
+    <changeSet author="keycloak" id="users-15699-drop-index">
+        <preConditions onSqlOutput="TEST" onFail="MARK_RAN">
+            <changeSetExecuted author="keycloak" id="users-1" changeLogFile="META-INF/users/jpa-users-changelog-1.xml" />
+        </preConditions>
+        <dropIndex tableName="kc_user_attribute" indexName="user_attr_name_value" />
+    </changeSet>
+
+    <changeSet author="keycloak" id="users-15699-drop-constraint" dbms="postgresql">
+        <preConditions onSqlOutput="TEST" onFail="MARK_RAN">
+            <changeSetExecuted author="keycloak" id="users-2" changeLogFile="META-INF/users/jpa-users-resource-changelog-1.xml" />
+        </preConditions>
+        <dropUniqueConstraint tableName="kc_user_attribute" constraintName="kc_user_attribute_fk_root_name_value_key" />
+    </changeSet>
+
+    <changeSet author="keycloak" id="users-15699-add-hash-column-index">
+        <ext:addGeneratedColumn tableName="kc_user_attribute">
+            <ext:column name="value_hash" type="kc_hash" function="kc_hash_function"/>
```

Comment:
I believe this should contain identification of column it hashes.
Can this be
- either used without `function="kc_hash_function"` part (could this be derived from `kc_hash` type)?
- or using a different xml tag, e.g. `ext:columnWithHash`?

Reply:
Thank you. I've explored the two suggestions deriving the column name from `kc_hash` type seems to me like not valid option, afaict the type does't have any information about the column it's using. Different xml tag would require yet another liquibase extension, we wouldn't be able to use the current one an it seems to me it'd bring unnecessary complexity. 

I've added a column identification into a `function` section. It is now in format of `function="kc_hash_function(value)"`. Please let me know what think.